--- a/esp32_p4_wifi6_touch_lcd_4_3.c
+++ b/esp32_p4_wifi6_touch_lcd_4_3.c
@@ -703,8 +703,6 @@ lv_display_t *bsp_display_start_with_config(bsp_display_cfg_t *cfg)
     assert(cfg != NULL);
     BSP_ERROR_CHECK_RETURN_NULL(esp_lv_adapter_init(&cfg->lv_adapter_cfg));
 
-    BSP_ERROR_CHECK_RETURN_NULL(bsp_display_brightness_init());
-
     BSP_NULL_CHECK(disp = bsp_display_lcd_init(cfg), NULL);
 
     BSP_NULL_CHECK(disp_indev = bsp_display_indev_init(cfg, disp), NULL);
